### PR TITLE
docs: minor typos

### PR DESCRIPTION
**Pull request by thesanjeevsharma**
Assuming by `users` we meant the end-user that will be using the software where testing is implemented and `method` here refers to JS method/function, I have updated two sentences.

Other Minor changes:
1. Capitalize **Queries**.
2. Full stop at the end of a sentence.
3. Changed 1(numeric) to one(text) to stay consistent.
4. Code-highlighted `div` and `span` as in the next point other HTML elements like `img`, `input`, etc were also highlighted.

### CODE REVIEW

File: docs/queries/about.mdx [hunk 1]
```diff
@@ -110,13 +110,14 @@ possible. With this in mind, we recommend this order of priority:
       `getByRole('button', {name: /submit/i})`. Check the
       [list of roles](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques#Roles).
    1. `getByLabelText`: Only really good for form fields, but this is the number
-      one method a user finds those elements, so it should be your top
-      preference.
+      one method to make sure that a user finds those elements, so it should be
```
Comment (alexkrolick) [inline docs/queries/about.mdx:113]: it's more like "users use labels to find form fields; this method emulates that"

Comment (thesanjeevsharma) [reply docs/queries/about.mdx:113]: Ah, okay!

I'll revert the changes. Are other minor changes legit?

Comment (alexkrolick) [reply docs/queries/about.mdx:113]: The change to getByText has the same issue.

Reading through the text, though, all of the phrasing could use a little work on the grammar. Open to updates.

File: docs/queries/about.mdx [hunk 2]
```diff
@@ -110,13 +110,14 @@ possible. With this in mind, we recommend this order of priority:
...
+      your top preference.
    1. `getByPlaceholderText`:
       [A placeholder is not a substitute for a label](https://www.nngroup.com/articles/form-design-placeholders/).
       But if that's all you have, then it's better than alternatives.
-   1. `getByText`: Not useful for forms, but this is the number 1 method a user
-      finds most non-interactive elements (like divs and spans).
+   1. `getByText`: Not useful for forms, but this is the number one method to
+      make sure that a user finds most non-interactive elements (like `div`s and
+      `span`s).
```
Comment (alexkrolick) [inline docs/queries/about.mdx:120]:
```suggestion
   1. `getByText`: Outside of forms, text content is the main way users find elements.
        This method can be used to find elements without other identifying attributes,
        like divs, spans, and paragraphs.
 ```

**Comment by MatanBobi** [comment]
@alexkrolick Are we good to merge this one?

**Comment by eps1lon** [comment]
@thesanjeevsharma Thanks!

@all-contributors Add @thesanjeevsharma for docs

**Pull request merged**